src/rpc/misc.cpp

```diff
@@ -323,11 +323,13 @@ UniValue createmultisig(const UniValue& params, bool fHelp)
 
 UniValue createwitnessaddress(const UniValue& params, bool fHelp)
 {
-    if (fHelp || params.size() < 1 || params.size() > 1)
+    if (fHelp || params.size() < 1 || params.size() > 1 || Params().NetworkIDString() == "main")
```

Comment:
Just not sure if that's useful for testing. Should we keep it for test only or remove it entirely? I don't have strong opinion.